frontend/README.md

```diff
@@ -0,0 +1,8 @@
+# React + Vite
+
+This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
```

Comment:
Pls update the file with steps on how to run the app locally. Also please a readme on the backend app also were you can explain how to bootstrap the app